dotcom-rendering/src/web/server/pageTemplate.ts

```diff
@@ -274,13 +299,20 @@ https://workforus.theguardian.com/careers/product-engineering/
 				</script>
 
 
-				${initTwitter}
+				${initTwitter ?? ''}
 
 
                 <noscript>
-                    <img src="https://sb.scorecardresearch.com/p?c1=2&c2=6035250&cv=2.0&cj=1&cs_ucfr=0&comscorekw=${encodeURIComponent(
-						keywords,
-					).replace(/%20/g, '+')}" />
+                    <img src="https://sb.scorecardresearch.com/p?${new URLSearchParams(
+						{
+							c1: '2',
+							c2: '6035250',
+							cv: '2.0',
+							cj: '1',
+							cs_ucfr: '0',
+							comscorekw: keywords,
+						},
+					).toString()}" />
```

Comment:
😍

Reply:
I thought this version was much nicer

Reply:
I had been meaning to refactor it for a while and this is a lot cleaner.